ArduCopter/ekf_check.cpp

```diff
@@ -200,12 +200,11 @@ void Copter::check_ekf_reset()
         AP::logger().Write_Event(LogEvent::EKF_YAW_RESET);
     }
 
-#if AP_AHRS_NAVEKF_AVAILABLE && HAL_NAVEKF2_AVAILABLE
-
+#if AP_AHRS_NAVEKF_AVAILABLE && (HAL_NAVEKF2_AVAILABLE || HAL_NAVEKF3_AVAILABLE)
     // check for change in primary EKF (log only, AC_WPNav handles position target adjustment)
```

Comment:
Comment's wrong - there's a call right here to poke the attitudecontrol

Reply:
ok thanks, updated.